widgets/common/src/DeployBlueprintModal.jsx

```diff
@@ -1,6 +1,5 @@
-/**
- * Created by kinneretzin on 05/10/2016.
- */
+const { i18n } = Stage;
+const t = (key, options) => i18n.t(`widgets.common.deployments.deployModal.${key}`, options);
```

Comment:
Optional:

Not sure about the naming of this function :smile: Seems a bit short to me. How about

```suggestion
const translate = (key, options) => i18n.t(`widgets.common.deployments.deployModal.${key}`, options);
```

Reply:
I was also thinking about that when I introduce this arrow function, but I think in this case (wrapper for `t` function from `i18n`) it is OK 🙂

Reply:
Sure deal :+1: let's skip it